<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HexEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
    <item>
     <widget class="QListWidget" name="o_fileList">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_empty">
       <attribute name="title">
        <string>Empty</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionEnable_Backup"/>
    <addaction name="actionRestore_File"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuBytes_per_row">
     <property name="title">
      <string>Bytes per row</string>
     </property>
     <addaction name="action16_2"/>
     <addaction name="action32"/>
     <addaction name="action64"/>
     <addaction name="action64_2"/>
    </widget>
    <addaction name="action_Copy"/>
    <addaction name="menuBytes_per_row"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuGroup_Size">
     <property name="title">
      <string>Group Size</string>
     </property>
     <addaction name="action1"/>
     <addaction name="action2"/>
     <addaction name="action4"/>
     <addaction name="action8"/>
     <addaction name="action16"/>
    </widget>
    <addaction name="action_Search"/>
    <addaction name="actionJump"/>
    <addaction name="separator"/>
    <addaction name="menuGroup_Size"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Search">
   <property name="text">
    <string>&amp;Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionEnable_Backup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Backup</string>
   </property>
  </action>
  <action name="actionJump">
   <property name="text">
    <string>Jump</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionRestore_File">
   <property name="text">
    <string>Restore File</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action16">
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action16_2">
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action32">
   <property name="text">
    <string>32</string>
   </property>
  </action>
  <action name="action64">
   <property name="text">
    <string>48</string>
   </property>
  </action>
  <action name="action64_2">
   <property name="text">
    <string>64</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
